card/lib/card/migration.rb

```diff
@@ -44,7 +44,8 @@ def schema_mode mig_type=type
 
       ActiveRecord::Base.table_name_suffix = new_suffix
       ActiveRecord::SchemaMigration.reset_table_name
-      yield
+      paths = Cardio.migration_paths(type)
+      yield(paths)
```

Comment:
no, this is to avoid repetition. In all blocks we passed to that method we called `Cardio.migration_paths`